examples/vitess/vtctld-pod.yaml

```diff
@@ -0,0 +1,37 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: vtctld
+  labels:
```

Comment:
please append a label that allows me to kill all vitess related pods/rcs/svc with kubectl delete po,rc,svc -l app=vitess